README.md

```diff
@@ -163,18 +181,18 @@ See [REST Documentation](docs/rest.md) for additional information
 
 To import scanned data into Event Portal:
```

Comment:
Indicate that the import process applies to stand-alone mode. Connected mode is automatic

Reply:
Done